lib/api/root.rb

```diff
@@ -152,6 +152,8 @@ def authorize_any(permissions, projects: nil, global: false, user: current_user)
       authenticate
     end
 
-    mount API::V3::Root
+    version 'v3', using: :path do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.